CONTRIBUTING.md

```diff
@@ -3,9 +3,10 @@
 ## Pull Requests
 We actively welcome pull requests.
 1. Fork the repo and create your branch from `develop`.
-2. If you've added code that should be tested, add tests.
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes.
+2. Apply code formatting by running ./scripts/apply-clang-format
```

Comment:
I don't know that I agree with that step.
I would only mention something like "Before sending your patch for review, please try to ensure that it is formatted properly.  We use clang-format version 8.0.0 for this."